### PR TITLE
Update README.md with mission statement

### PR DESCRIPTION
**Pull request by MichaelLyst**
Updates the readme to have a mission statement about why this exists.

### CODE REVIEW

**Comment by mihazelnik** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+
+This repository is an extension of our other work as MakingLyst - a chance to peek into the tech and design at Lyst. Or, to put it another way, a chance for us to be transparent about how we work, why we work the way we do and, to give back to the community through sharing of ideas and code.
+
+In here you will find how to's, guides, code snippets and anything else that we find helpful and that we don't need to hide away on our internal Wiki.
+
+We are based in London, England and we welcome your feedback, comments and, yes, pull requests.
+
+You can read our blog here http://developers.lyst.com/
+
+You can ready our tweets here https://twitter.com/makinglyst
```
Comment (trepca) [inline README.md]: typo

**Comment by phalt** [comment]
Nice. I should move the API design stuff to here.

**Comment by phalt** [comment]
:+1:

**Pull request merged**